gammapy/cube/psf_map.py

```diff
@@ -135,12 +156,25 @@ def geom(self):
     @classmethod
     def read(cls, filename, **kwargs):
         """Read a psf_map from file and create a PSFMap object"""
-        psfmap = Map.read(filename, **kwargs)
-        return cls(psfmap)
-
-    def write(self, *args, **kwargs):
+        with fits.open(filename, memmap=False) as hdulist:
+            psfmap = Map.from_hdulist(hdulist, "PSFMAP", "BANDSPSF", "auto")
+            if "EXPMAP" in hdulist:
+                expmap = Map.from_hdulist(hdulist, "EXPMAP", "BANDSEXP", "auto")
+            else:
+                expmap = None
+        return cls(psfmap, expmap)
+
+    def write(self, filename, overwrite=False):
         """Write the Map object containing the PSF Library map."""
```

Comment:
Docstring isn't very precise or useful. Suggest to either just put "Write to FITS file" (that's what I would do), or to explain what is written.